www/zemk05/SP1/charity.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>LUSH | Charita</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/charity.css" />
```

Comment:
kazda stranka ma vlastni styl? to neni best practice, v ramci jednoho webu je dobre mit jeden spolecny soubor se styly, urcite jsi prisla na to, ze nektere styly se opakujou v tech CSS souborech

Reply:
jojo já to dělám záměrně, protože je to pro mě přehlednější při práci kvůi objemu toho csska, pak to dám do jednoho, nevadí to? lépe se mi pracuje

Reply:
ale klidně to dám do jednoho, prostě si to tam jen komenty nějak rozdělím

Reply:
jo to bude lepsi, protoze se bojim ze by ses v tom ztratila